calcom/cal.com
# Add destination calendar name to DestinationCalendarSelector

## What does this PR do?



This PR adds the destination calendar name to the DestinationCalendarSelector component when indicating it is using the default destination calendar.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/65426560/214606657-273ee5a1-712e-45d7-a31f-192335f8338c.png">


Fixes #6634



**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Set your destination calendar to a calendar that is not the primary one for that acount
- Go to an event type that is using the destination calendar. It should display the name of the selected destination calendar

## Checklist



- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
